onchain/rollups/contracts/inputs/IInputBox.sol

```diff
@@ -7,26 +7,26 @@ pragma solidity ^0.8.8;
 interface IInputBox {
     /// @notice Emitted when an input is added to a DApp's input box.
     /// @param dapp The address of the DApp
-    /// @param inputIndex The index of the input in the input box
+    /// @param index The index of the input in the input box
     /// @param sender The address that sent the input
-    /// @param input The contents of the input
+    /// @param payload The input payload
     /// @dev MUST be triggered on a successful call to `addInput`.
     event InputAdded(
         address indexed dapp,
-        uint256 indexed inputIndex,
+        uint256 indexed index,
         address sender,
-        bytes input
+        bytes payload
```

Comment:
Maybe it's wise to keep this parameter being name `input`, because it is no longer the `payload` provided to the `addInput` function. Also, we may need to make this distinction clear in the documentation of the `IInputBox` interface.

Reply:
I think it's more gas efficient to emit the payload rather than the whole input blob. Because `index` is already emitted and `block.number`,`block.timestamp` are retrievable as event meta data  too.

Reply:
True, let's discuss this with the node team.

Reply:
I see that @gligneul agreed with the event format as (`dapp`,`index`,`input`). I will make the changes